query_optimizer/resolver/Resolver.cpp

```diff
@@ -702,6 +702,7 @@ StorageBlockLayoutDescription* Resolver::resolveBlockProperties(
   } else if (type_string.compare("columnstore") == 0) {
     description->set_sub_block_type(
         quickstep::TupleStorageSubBlockDescription::BASIC_COLUMN_STORE);
+    // NOTE(zuyu): sort is optional.
```

Comment:
The variable name ``block_requires_sort`` is confusing, because the basic column store block doesn't really "require" sorting. You may rename the variable to ``block_allows_sort`` or something clearer.